pennylane_lightning/src/tests/VectorStream.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
```

Comment:
The original purpose I remember is that one in `Util.hpp` is inside `Pennylane::Util` namespace (instead of global) and I want to reduce dependencies in a test code. Still, as it seems like we do not use this file at the moment, I think we may remove it.